src/cursor.rs

```diff
@@ -0,0 +1,167 @@
+use std::{
```

Comment:
Unfortunately git didn't recognize this as a move of the impatient module file, so it shows everything as new. I've pointed out the changes below.